doc/classes/ProjectSettings.xml

```diff
@@ -868,11 +868,29 @@
 		<member name="debug/shapes/navigation/3d/link_connection_disabled_color" type="Color" setter="" getter="" default="Color(0.5, 0.5, 0.5, 1)">
 			Color to use to display disabled navigation link connections, visible when "Visible Navigation" is enabled in the Debug menu.
 		</member>
-		<member name="debug/shapes/paths/geometry_color" type="Color" setter="" getter="" default="Color(0.1, 1, 0.7, 0.4)">
-			Color of the curve path geometry, visible when "Visible Paths" is enabled in the Debug menu.
+		<member name="debug/shapes/paths/2d/fish_bones_enabled" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], [Path2D] curve paths will by default add additional fish bones along the path segments to visualize direction.
 		</member>
-		<member name="debug/shapes/paths/geometry_width" type="float" setter="" getter="" default="2.0">
-			Line width of the curve path geometry, visible when "Visible Paths" is enabled in the Debug menu.
+		<member name="debug/shapes/paths/2d/fish_bones_interval" type="int" setter="" getter="" default="4">
+			The default [Path2D] curve path sample point interval in which fish bones are places along the path. E.g. a value of 4 places a fish bone for every 4-th sampled point along path.
+		</member>
+		<member name="debug/shapes/paths/2d/geometry_color" type="Color" setter="" getter="" default="Color(0.1, 1, 0.7, 0.4)">
+			Color of the [Path2D] curve path geometry, visible when "Visible Paths" is enabled in the Debug menu.
```

Comment:
Something like this? I'm not sure it's accurate, or maybe it's too many generic "_used_":
```suggestion
			The color used to display the curve of a path, for [Path2D] nodes. Used when "Visible Paths" is enabled in the Debug menu.
```
or
```suggestion
			The color used to display a path, for [Path2D] nodes. Used when "Visible Paths" is enabled in the Debug menu.
```
or similar?